experiments/tool/pmd-bin-5.2.0/bin/runTestSubjects.sh

```diff
@@ -1,94 +0,0 @@
-#!/bin/bash
-
-echo "Starting script..."
```

Comment:
This script looks fine, but is this the only script you wrote? Are there other things in this commit that you actually wrote? If there are other things please consider splitting the waterloo tool into a commit where it contains the stuff you wrote and a commit that contains the just the files for the tool (unedited).